### PR TITLE
Issue 274: Improve documentation on environment variables

### PR DESCRIPTION
**Pull request by damien-carcel**
## Description

This PR adds missing documentation about environment variables available in the different images, and how to use them to configure `composer` and `xdebug`.

Only the variable `PHP_IDE_CONFIG` is not handled here, as it will be in #174.

## Definition Of Done

| Q                                 | A
| --------------------------------- | ---
| Added tests                       | -
| Changelog updated                 | -
| Documentation                     | OK
| Fixed tickets                     | #274

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: Docs/symfony/docker-compose.yml.apache_dist [hunk 1]
```diff
@@ -8,7 +8,7 @@ services:
       PHP_IDE_CONFIG: 'serverName=your-key'           # Mandatory for debugging
       PHP_XDEBUG_ENABLED: 0                           # [optional] This is the default value, change to 1 to enable Xdebug
       PHP_XDEBUG_IDE_KEY: XDEBUG_IDE_KEY              # [optional] This is the default value
-      PHP_XDEBUG_REMOTE_HOST: xxx.xxx.xxx.xxx         # Replace with you host IP (only for Windows and Mac as Linux uses socket)
+      PHP_XDEBUG_REMOTE_HOST: xxx.xxx.xxx.xxx         # Replace with you host IP (only for Windows and Mac, as Linux uses IPC socket)
```
Comment (tamarasaurus) [inline Docs/symfony/docker-compose.yml.apache_dist]: 'you' should be 'your' right?

File: Docs/symfony/fpm.md [hunk 2]
```diff
@@ -2,9 +2,28 @@
 
 ## Using the images
 
-The simplest way to run the containers is to copy the [FPM compose file](https://github.com/akeneo/Dockerfiles/blob/master/Docs/symfony/docker-compose.yml.fpm_dist) at the root of your project (don't forget to rename it `docker-compose.yml`).
+Just copy the [FPM compose file](https://github.com/akeneo/Dockerfiles/blob/master/Docs/symfony/docker-compose.yml.fpm_dist) at the root of your project (don't forget to rename it `docker-compose.yml`).
```
Comment (tamarasaurus) [inline Docs/symfony/fpm.md]: 'to the root'

File: Docs/symfony/fpm.md [hunk 3]
```diff
@@ -2,9 +2,28 @@
...
+    volumes:
+      - ./:/srv/application
+
+  nginx:
+    image: nginx
+    volumes:
+      - ./:/srv/application:ro
+```
+
+Read only access is enough for nginx, only FPM need a full access on the application.
```
Comment (tamarasaurus) [inline Docs/symfony/fpm.md]: 'needs full access'

File: Docs/symfony/fpm.md [hunk 4]
```diff
@@ -2,9 +2,28 @@
...
+
+**Important**: As the application is shared between your host and the FPM container, it is mandatory on Linux that your user ID and group ID are `1000`,
+as it is for the `docker` user inside the container. Without that, you'll end up with a permission mess that will prevent your application to run.
```
Comment (tamarasaurus) [inline Docs/symfony/fpm.md]: to run -> from running

File: Docs/symfony/fpm.md [hunk 5]
```diff
@@ -73,10 +92,54 @@ $ docker-compose exec fpm composer update
...
+sharing both your configuration (including your GitHub token, which is mandatory to install Akeneo PIM standard and/or enterprise edition)
+and your composer cache (and so reducing considerably the dependencies installation time):
+```yaml
+services:
+  fpm:
+    volumes:
+      - ~/.composer:/home/docker/.composer
+```
+
+Beware than on recent Linux system, `composer` cache and configuration are usually separated in `.cache/composer` and `.config/composer`.
```
Comment (tamarasaurus) [inline Docs/symfony/fpm.md]: Beware that

File: Docs/symfony/mod_php.md [hunk 6]
```diff
@@ -2,9 +2,21 @@
...
+services:
+  apache:
+    image: akeneo/apache-php
+    user: docker
+    volumes:
+      - ./:/srv/application
+```
+
+**Important**: As the application is shared between your host and the FPM container, it is mandatory on Linux that your user ID and group ID are `1000`,
+as it is for the `docker` user inside the container. Without that, you'll end up with a permission mess that will prevent your application to run.
```
Comment (tamarasaurus) [inline Docs/symfony/mod_php.md]: 'from running'

**Review by tamarasaurus — APPROVED** [review]

File: Docs/symfony/docker-compose.yml.apache_dist (hunk 1 above)
Comment (damien-carcel) [reply Docs/symfony/docker-compose.yml.apache_dist]: Oups, indeed ^^, thank you

**Pull request merged**